bc_obps/common/fixtures/dashboard/bciers/internal.json

```diff
@@ -32,18 +32,18 @@
             ]
           },
           {
-            "title": "Registration",
+            "title": "Transfers",
             "href": "/registration",
```

Comment:
I think this href line links us to the sub-dashboard with the same name. If we don't want a transfers sub-dashboard (?), maybe we can cut this or do ""

Reply:
I don't think it's clear from the wireframes whether we expect a "Registration"/"Transfers" sub-dashboard for internal users. For the time being I chose to leave it in, on the weak grounds that the breadcrumbs in this wireframe show that the Transfer grid page comes underneath the "Registration" breadcrumb

https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=20612-5693&m=dev&t=JgYELMKg7eWLmaHN-1

so we've now got this Registration sub-dashboard (that IMO serves no purpose, but whatever)
![Screenshot 2024-10-29 at 6 00 38 PM](https://github.com/user-attachments/assets/ce5f1eac-6480-4d2a-bd29-4ff683149341)

and clicking on that tile takes us to the eventual home of the Transfer grid page
![Screenshot 2024-10-29 at 6 01 45 PM](https://github.com/user-attachments/assets/ee20a2d7-10ed-47d9-ad38-69cbc5ba0f2b)

Reply:
This comment made me laugh and accidentally spit out my bagel, I hope you're happy with yourself

Reply:
...but it wasn't even a funny comment...